components/script/dom/document.rs

```diff
@@ -456,6 +457,15 @@ pub struct Document {
     #[no_trace]
     #[ignore_malloc_size_of = "AnimationTickType contains data from an outside crate"]
     pending_animation_ticks: DomRefCell<AnimationTickType>,
+    /// <https://drafts.csswg.org/resize-observer/#dom-document-resizeobservers-slot>
+    ///
+    /// Note: we are storing, but never removing, resize observers.
+    /// The lifetime of resize observers is specified at
+    /// https://drafts.csswg.org/resize-observer/#lifetime.
+    /// But implementing it comes with known problems:
+    /// - https://bugzilla.mozilla.org/show_bug.cgi?id=1596992
+    /// - https://github.com/w3c/csswg-drafts/issues/4518.
```

Comment:
URLs need to be wrapped in rustdoc comments or it will cause a warning:

```suggestion
    /// <https://drafts.csswg.org/resize-observer/#lifetime>.
    /// But implementing it comes with known problems:
    /// - <https://bugzilla.mozilla.org/show_bug.cgi?id=1596992>
    /// - <https://github.com/w3c/csswg-drafts/issues/4518>
```